payslip_line_manually/models/hr_salary_rule.py

```diff
@@ -0,0 +1,31 @@
+from odoo import fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class HrSalaryRule(models.Model):
+    _inherit = "hr.salary.rule"
+
+    analytic_account_id = fields.Many2one(
+        "account.analytic.account", "Analytic Account"
+    )
+    line_manually_model = fields.Selection(
+        [("hr.contract", "Contract"), ("hr.payslip", "Payslip")],
+        string="Salary Rule input from",
+    )
+
+    def _compute_values_python(self, localdict):
```

Comment:
The same here, if you dont call super() you are overriding and then not supporting other modules inheriting this.